web/src/routes/storage.tsx

```diff
@@ -61,6 +62,11 @@ const routes = (): Route => ({
       element: <ISCSIPage />,
       handle: { name: N_("iSCSI") },
     },
+    {
```

Comment:
add it only conditionally if dasd is supported ( will do it myself ).

Reply:
How did you solve this?

Reply:
I thought that @teclator did it with that new supported call that should do it.

Reply:
We only added it in the selector, showing the option in case that supported but not in the routes, then you commented it here, but we can leave it as it is as the button for navigating to the route will not be shown although modifying the url will allow to navigate there even if not supported

Reply:
Done by https://github.com/openSUSE/agama/pull/1549/commits/a75354f1ddb8f9eff6a54689e75f69d15d9860fa